Week2/homework/maartjes-work.js

```diff
@@ -1,65 +1,71 @@
 'use strict';
 
 const monday = [
-  {
-    name: 'Write a summary HTML/CSS',
-    duration: 180,
-  },
-  {
-    name: 'Some web development',
-    duration: 120,
-  },
-  {
-    name: 'Fix homework for class10',
-    duration: 20,
-  },
-  {
-    name: 'Talk to a lot of people',
-    duration: 200,
-  },
+	{
+		name: 'Write a summary HTML/CSS',
+		duration: 180,
+	},
+	{
+		name: 'Some web development',
+		duration: 120,
+	},
+	{
+		name: 'Fix homework for class10',
+		duration: 20,
+	},
+	{
+		name: 'Talk to a lot of people',
+		duration: 200,
+	},
 ];
 
 const tuesday = [
-  {
-    name: 'Keep writing summary',
-    duration: 240,
-  },
-  {
-    name: 'Some more web development',
-    duration: 180,
-  },
-  {
-    name: 'Staring out the window',
-    duration: 10,
-  },
-  {
-    name: 'Talk to a lot of people',
-    duration: 200,
-  },
-  {
-    name: 'Look at application assignments new students',
-    duration: 40,
-  },
+	{
+		name: 'Keep writing summary',
+		duration: 240,
+	},
+	{
+		name: 'Some more web development',
+		duration: 180,
+	},
+	{
+		name: 'Staring out the window',
+		duration: 10,
+	},
+	{
+		name: 'Talk to a lot of people',
+		duration: 200,
+	},
+	{
+		name: 'Look at application assignments new students',
+		duration: 40,
+	},
 ];
 
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+	const inHoursArr = tasks.map(hours => task.duration / 60).filter(hours => hours <= 2);
+	const maartjesPayArr = inHoursArr.map(hours => hours * maartjesHourlyRate);
+	const totalPay = maartjesPayArr.reduce((sum, amount) => sum + amount);
+
+	console.log(totalPay);
+	return totalPay;
+
+
 }
 
-// eslint-disable-next-line no-unused-vars
+// eslint - disable - next - line no - unused - vars
```

Comment:
Hmm, what's going on here?

Reply:
Half of an exercise and from week 2, work in progress!